src/components/TheNavBar.vue

```diff
@@ -134,15 +194,27 @@
     data() {
       return {
         showNav: false,
+        showAboutModal: false,
+        appName: "",
+        appVersion: "",
       }
     },
+    mounted() {
+      this.appName = this.$appName
+      this.appVersion = require("../../package.json").version
```

Comment:
I'd rather have this logic outside of the component and in `main.ts`. You can probably create a `$appVersion` global value.